AsyncDisplayKit/ASCellNode.m

```diff
@@ -123,7 +123,6 @@ - (void)setText:(NSString *)text
   _textNode.attributedString = [[NSAttributedString alloc] initWithString:_text
                                                                attributes:@{NSFontAttributeName: [UIFont systemFontOfSize:kFontSize]}];
 
-  [self invalidateCalculatedLayout];
```

Comment:
-invalidateCalculatedLayout is removed here because calling -invalidateCalculatedLayout here removes the current constrained size and therefore any -setNeedsLayout calls in the future won't have a valid constrained size to proceed. Instead, cell nodes should be relaid-out using the new APIs introduced in ASTableView and ASCollectionView. 

The commit that makes this removal (91f3ba1) has the same explanation, in case people do a git blame later on :)